packages/ocular/src/approaches/chat-retrieve-read-retrieve.ts

```diff
@@ -79,68 +81,65 @@ export default class ChatReadRetrieveRead implements IChatApproach {
   // }
 
   async run(messages: Message[], context?: ChatContext): Promise<ChatResponse> {
-    const { completionRequest, thoughts } = await this.baseRun(
-      messages,
-      context
-    );
-    let hits = [];
-    const chatCompletion = await this.openaiService_.completeChat(
-      completionRequest.messages
-    );
-    return {
-      // choices: [
-      //   {
-      //     index: 0,
-      //     message: {
-      //       content: chatCompletion,
-      //       role: 'assistant',
-      //       context: {
-      //         data_points: hits,
-      //         thoughts: thoughts,
-      //       },
-      //     },
-      //   },
-      // ],
-      message: {
-        role: "assistant",
-        content: chatCompletion,
-      },
-      data_points: hits,
-    };
+    // const { completionRequest, thoughts } = await this.baseRun(
```

Comment:
@Vivek-Lahole do we still need this uncommented block?

Reply:
yes , need to clean that up for a non-streaming response